homeassistant/components/tailscale/const.py

```diff
@@ -12,3 +12,5 @@
 SCAN_INTERVAL = timedelta(minutes=1)
 
 CONF_TAILNET: Final = "tailnet"
+
+AUTHKEYS_URL = "https://login.tailscale.com/admin/settings/keys"
```

Comment:
If this constant is only used in `config_flow.py`, please place it in `config_flow.py`.

Reply:
Thanks for the heads up. Already replaced it.